### DISCUSSION TITLE
V cycle

### DISCUSSION THREAD

**Discussion by shadowbourne** [wenceorg/comp4187#102] [Q&A] [answered]
When solving at the coarsest level, are we allowed/supposed to use lu_solve? Or should we use additional jacobi relaxations as our "exact solve"?

thanks!

**Comment by annereinarz** [comment, marked as answer]
I would recommend using the lu_solve at the coarsest level. For some types of problems using a few iterations of a smoother is sufficient, but the direct solve is the "safer" option.

**Reply by shadowbourne** [reply]
Okay thankyou!

Additionally, my v cycle gets down to errors of 1e-9 and then begins to plateau from arithmetics errors etc on the test_one_dim mms tests (passes them all)

However, when running on 1a) (for 3a)), I manage to get the convergence order to = 2.5 for the first 4 grid refinements (from grid of 5 as starting point), with error going down to 1e-5 or 6, but then with further grid refinements the error explodes (to 1e25 etc)?

Is this expected / due to the lack of smoothness ? Or is something wrong here?

Thanks.

**Reply by annereinarz** [reply]
The plateauing behavior is quite typical. The explosion of the error is a unexpected. I haven't seen the same, but I would also not to worry about it overly much. It may be down to the exact parameters used. The theoretical results from the lectures do not hold here (as you say there is a lack of smoothness).
